src/main/scala/com/fulcrumgenomics/umi/CorrectUmis.scala

```diff
@@ -260,6 +262,14 @@ class CorrectUmis
       if (wrongLengthRecords > 0) logger.error(s"# ${wrongLengthRecords} had unexpected UMIs of differing lengths in the BAM file!")
       logger.error("###################################################################")
     }
+
+    minCorrected.foreach { min =>
+      val ratioKept = 1.0 * kept / totalRecords
+      validate(ratioKept >= min,
```

Comment:
Validation exceptions are special for the constructor, validation command line parameters.  Sorry if this is not explicit anywhere.  How about assert?